Userland/Libraries/LibWeb/CSS/StyleComputer.cpp

```diff
@@ -1870,6 +1885,47 @@ RefPtr<Gfx::Font const> StyleComputer::compute_font_for_style_values(DOM::Elemen
         found_font = find_font(font_family.as_custom_ident().custom_ident());
     }
 
+    // 5.2.6 If no matching face exists or the matched face does not contain a glyph for the character to be
+    // rendered, the next family name is selected and the previous three steps repeated. Glyphs from other
+    // faces in the family are not considered. The only exception is that user agents may optionally substitute
+    // a synthetically obliqued version of the default face if that face supports a given glyph and synthesis
+    // of these faces is permitted by the value of the font-synthesis property. For example, a synthetic italic version
+    // of the regular face might be used if the italic face doesn’t support glyphs for Arabic.
+    auto face_contains_glyphs = [&](RefPtr<Gfx::Font const> font, FlyString const& text) -> bool {
```

Comment:
The places that call this always pass a non-null font, so it could be `Gfx::Font const& font` instead. Then you wouldn't need the extra `if  (font)` check below. (Just dereference `*` the font when calling this lambda.)

Reply:
Thank you for the recommendation! I have done the changes to use a reference instead of the RefPtr.